gnd/cosmos/build_cosmos_plugin.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I am wondering if this might be better stored with Adamant instead of the example